ingestion-sink-release fails on master

Looks like this CI job invokes a container that doesn't contain the beam pom, but does expect it to be there. Perhaps it's not getting the right profile activated.

See https://circleci.com/gh/mozilla/gcp-ingestion/12923?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification

> sink_1  | [ERROR] Child module /app/ingestion-beam of /app/pom.xml does not exist @

--- a/ingestion-sink/docker-compose.yml
+++ b/ingestion-sink/docker-compose.yml
@@ -7,4 +7,4 @@ services:
       dockerfile: ingestion-sink/Dockerfile
     image: &image "${DOCKER_IMAGE_PREFIX}ingestion-sink:${CIRCLE_TAG-latest}"
     command:
-      mvn clean test
+      mvn -P ingestion-sink clean test

